CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.2.1
```

Comment:
Super-nit: while 1.2.1 is overwhelmingly likely to be the next version, my suggestion would be to accumulate changes under "Unreleased changes" and change the heading to "1.2.1" in the release commit.

```suggestion
# Unreleased changes
```

Reply:
Thanks btw for adding this to the changelog - something I forgot to do in my PR.